src/main/scala/ReleaseExtra.scala

```diff
@@ -225,7 +225,7 @@ object ReleaseStateTransformations {
   }
 
   lazy val publishArtifacts = ReleaseStep(
-    action = publishArtifactsAction,
+    action = runPublishArtifactsAction,
```

Comment:
@davidhoyt @gseitz The related changes here appear to have broken publishing unsigned artifacts between 0.8.4 and 0.8.5.